Sources/Msg.php

```diff
@@ -1719,6 +1714,14 @@ public static function modify(array &$msgOptions, array &$topicOptions, array &$
 			$searchAPI->postRemoved((int) $msgOptions['id']);
 		}
 
+		// If this is the first post of a topic, remove any cached slug string for the topic.
+		if (
+			!empty($topicOptions['id'])
+			&& ($msgOptions['id'] ?? NAN) === ($topicOptions['first_msg'] ?? NAN)
+		) {
+			CacheApi::put('slug_type-topic_id-' . $topicOptions['id'], null, 0);
```

Comment:
You forgot to add `use SMF\Cache\CacheApi;`

Reply:
Oops! Fixed now.